crates/services/p2p/src/codecs/postcard.rs

```diff
@@ -274,13 +274,11 @@ impl ProtocolName for MessageExchangePostcardProtocol {
 
 #[cfg(test)]
 mod tests {
-    use fuel_core_types::blockchain::primitives::BlockId;
-
     use super::*;
 
     #[test]
     fn test_request_size_fits() {
-        let m = RequestMessage::Transactions(BlockId::default());
+        let m = RequestMessage::Transactions(2..6);
```

Comment:
I like defining values like this as variables in tests, just to make it easier to read:
```
let arbitrary_range = 2..6
```
if it's just an arbitrary value.